db/seeds/items.rb

```diff
@@ -1,10 +1,10 @@
 category = Category.find_by_name("T-shirts")
-category.items.create(name: "Mens  T-shirt", description: "100% Cotton", image_url: "http://flathash.com/fake_url_01", price: 10.43)
-category.items.create(name: "Womens T-shirt", description: "100% Cotton", image_url: "http://flathash.com/fake_url_02", price: 11.43)
+category.items.create(name: "Men's  T-shirt", description: "100% Cotton", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/images/mens_t-shirt--420x280.jpg", price: 10.43)
+category.items.create(name: "Women's T-shirt", description: "100% Cotton", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/images/womens_t-shirt--420x280.jpg", price: 11.43)
 
-category = Category.find_by_name("HouseHold")
-category.items.create(name: "Mug", description: "100% Ceramic", image_url: "http://flathash.com/fake_url_03", price:4.43 )
+category = Category.find_by_name("Household Items")
+category.items.create(name: "Mug", description: "100% Ceramic", image_url: "https://s3-us-west-2.amazonaws.com/dadjokesfordays/images/mug_with_MUG_joke--420x280.jpg", price:4.43 )
```

Comment:
Line is too long. [179/80]<br>Space missing after colon.<br>Space inside parentheses detected.